test/blocks_test.rb

```diff
@@ -1256,11 +1256,20 @@ def names
 ++++
       EOS
 
-      expect = <<-'EOS'
+      expect = begin
+        gem 'asciimath', '~> 1.0.0.a'
```

Comment:
Can you upgrade this to 1.0.0.preview.1? Though I'd be okay with adding it as a development dependency so we can run the test without the begin/rescue.